roles/rhn-subscription-osd/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Register with RHN
+  redhat_subscription:
+    state: present
+    username: "{{ redhat.user }}"
+    password: "{{ redhat.pass }}"
+
+- name: Pause for registration
```

Comment:
Pause is only needed for very first run when we actually registering to rhn for the very first time. For repetitive ursula run might be good to avoid 90sec pause.